### PR TITLE
[Loyalty QA] center log in button text in safari

### PR DESCRIPTION
**Pull request by xtina-starr**
closes: https://github.com/artsy/reaction-force/issues/103

By default, the `flex-direction` property is set to `row` which aligns the flex-items along the main (x) axis. It looks like that is what's causing the "LOG IN" text to sit to the left. I think we want to style it to have`flex-direction: column` in this case. 

screen shot 2017-04-18 at 1 12 47 pm

### CODE REVIEW

File: src/apps/loyalty/containers/login/index.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Login extends React.Component<LoginProps, LoginState> {
 
           {form.csrfToken && <input type="hidden" name="_csrf" value={form.csrfToken} />}
 
-          <Button block>Log In</Button>
+          <Button block style={{flexDirection: "column"}}>Log In</Button>
```
Comment (l2succes) [inline src/apps/loyalty/containers/login/index.tsx]: can you add that style to the base button component instead?

Comment (xtina-starr) [inline src/apps/loyalty/containers/login/index.tsx]: The facebook and twitter buttons need to be set to the default which is why I added it here but I can still move it to the base button component and set the `flex-direction` to row for the twitter and facebook button components

**Comment by xtina-starr** [comment]
I anticipate my last commit will fail the build because of the snap shot. How do I go about updating that?

**Comment by l2succes** [comment]
@xtina-starr run `yarn test -- -u` to update snapshots.

**Comment by l2succes** [comment]
Looks good! We can merge after updating the snapshot tests

**Comment by xtina-starr** [comment]
tests are 💚  gonna go ahead and merge.

**Pull request merged**